### PR TITLE
[CHAD-8813] Updated 'qubino-flush-thermostat' and 'qubino-flush-thermostat-cooling'

### PR DESCRIPTION
**Pull request by Bhavik-T**
Updated 'qubino-flush-thermostat' and 'qubino-flush-thermostat-cooling' to have temperature range of (-13F to 176F)

### CODE REVIEW

**Review by greens — APPROVED** [review]

**Comment by iplantemn** [comment]
@bflorian @greens updated capability configurations

File: drivers/SmartThings/zwave-thermostat/profiles/qubino-flush-thermostat-cooling.yml [hunk 1]
```diff
@@ -6,6 +6,10 @@ components:
     version: 1
   - id: thermostatCoolingSetpoint
     version: 1
+    configuration:
```
Comment (tpmanley) [inline drivers/SmartThings/zwave-thermostat/profiles/qubino-flush-thermostat-cooling.yml:9]:
```suggestion
    config:
```

File: drivers/SmartThings/zwave-thermostat/profiles/qubino-flush-thermostat.yml [hunk 2]
```diff
@@ -6,6 +6,10 @@ components:
     version: 1
   - id: thermostatHeatingSetpoint
     version: 1
+    configuration:
```
Comment (tpmanley) [inline drivers/SmartThings/zwave-thermostat/profiles/qubino-flush-thermostat.yml:9]:
```suggestion
    config:
```

**Comment by greens** [comment]
@KKlimczukS I think this should be correct now?

File: drivers/SmartThings/zwave-thermostat/profiles/qubino-flush-thermostat-cooling.yml [hunk 3]
```diff
@@ -6,6 +6,10 @@ components:
     version: 1
   - id: thermostatCoolingSetpoint
     version: 1
+    config:
+      values:
+        - key: "coolingSetpoint.value"
+          range: [ -10, 80 ]
```
Comment (ctowns) [inline drivers/SmartThings/zwave-thermostat/profiles/qubino-flush-thermostat-cooling.yml:12]: Shouldn't these be [-25, 80C] to match the requested [-13, 176F] range from the DTH?
https://smartthings.atlassian.net/browse/CHAD-8813?focusedCommentId=1224688

Comment (greens) [reply drivers/SmartThings/zwave-thermostat/profiles/qubino-flush-thermostat-cooling.yml]: should be fixed

**Review by ctowns — APPROVED** [review]

**Pull request merged**